### DISCUSSION TITLE
2010/11/16/%E7%BF%BB%E5%A2%99%E8%BD%AF%E4%BB%B6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#302] [Announcements]
# 2010/11/16/%E7%BF%BB%E5%A2%99%E8%BD%AF%E4%BB%B6

大概只有中国才会产生这么多五花八门的翻墙工具。最近接触了几款，对它们最深的印象是：名字真好听，浸润了浓郁的中国文化韵味。

https://blog.qizhen.xyz/2010/11/16/%E7%BF%BB%E5%A2%99%E8%BD%AF%E4%BB%B6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2010年12月3日 15:57 编辑
我正在研究看有什么方法可以自动同步内容呢。

回复
mangguopai
2010年12月2日 20:38 编辑
强烈要求转移Blog到Sina。WordPress墙了啊

回复
foolooo
2010年11月25日 13:10 编辑
话说阮老师在墙另外一边讲翻墙的方法，意义不大啊～

回复
